src/spec/index.ts

```diff
@@ -15,7 +15,7 @@ import {TopLevel} from './toplevel.js';
 import {FacetedUnitSpec, GenericUnitSpec, NormalizedUnitSpec, TopLevelUnitSpec, UnitSpecWithFrame} from './unit.js';
 
 export type {BaseSpec, LayoutSizeMixins} from './base.js';
-export type {GenericHConcatSpec, GenericVConcatSpec, NormalizedConcatSpec} from './concat.js';
+export type {GenericConcatSpec, GenericHConcatSpec, GenericVConcatSpec, NormalizedConcatSpec} from './concat.js';
```

Comment:
Noting `GenericConcatSpec` is a new export, but that seems reasonable!

Reply:
Yeah